### PR TITLE
Added fade animation feature on MissionPillar, About, and Event. Text…

### LINKED ISSUE #128: Add animation
### Description

Adding animation to the page to make it more engaging

### Solution description

Install and use react-reveal on the page.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Zackius**
… border bottom in componenets headers

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by orama254** [comment]
Hello @Zackius 

Your preview build seems to be failing on our end. Are the packages resolved correctly on your end?

**Comment by Zackius** [comment]
I looked at the issue, packages are resolved.

**Review by orama254 — COMMENTED** [review]
@Zackius 
The error happens on the event component as well. Check it out

PATH: /src/components/Events/Events.tsx

**Comment by Zackius** [comment]
Error resolved on Event file.

File: src/components/AboutUs/AboutUs.tsx
Comment (orama254) [inline src/components/AboutUs/AboutUs.tsx:1]: I can see an additional underline on the h2, could you have that resolved so as to maintain the original design specifications

File: src/components/ContactUs/DesktopVersion.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import ReCAPTCHA from 'react-google-recaptcha';
 export default function DesktopVersion() {
   const [captchaComplete, setCaptchaComplete] = useState<boolean>(false);
 
-  const [state, handleSubmit] = useForm(process.env.NEXT_PUBLIC_FORMSPREE_ID!);
+  const [state, handleSubmit] = useForm("process.env.NEXT_PUBLIC_FORMSPREE_ID!");
```
Comment (orama254) [inline src/components/ContactUs/DesktopVersion.tsx]: remove the extra "" marks from the environment variable declaration.

File: src/components/ContactUs/DesktopVersion.tsx [hunk 2]
```diff
@@ -84,7 +84,7 @@ export default function DesktopVersion() {
           />
 
           <ReCAPTCHA
-            sitekey={process.env.NEXT_PUBLIC_RECAPTCHA_SITE_KEY!}
+            sitekey={"process.env.NEXT_PUBLIC_RECAPTCHA_SITE_KEY!"}
```
Comment (orama254) [inline src/components/ContactUs/DesktopVersion.tsx]: remove the extra "" marks from the environment variable declaration.

File: src/components/Events/Events.tsx [hunk 3]
```diff
@@ -60,10 +61,12 @@ export default function Events() {
       id="events"
       className="relative flex flex-col items-center justify-center w-full py-16 bg-white md:py-20 gap-y-2 md:gap-y-5"
     >
-      <h2 className="flex items-center justify-center font-light tracking-wide">
+      <Fade direction='right'>
```
Comment (orama254) [inline src/components/Events/Events.tsx]: Could you the fade direction to down

File: src/components/HeroHeader/DesktopHero.tsx
Comment (orama254) [inline src/components/HeroHeader/DesktopHero.tsx:1]: For the hero section, could you add the fade direction to down?

File: src/components/MissionPillars/MissionPillars.tsx [hunk 4]
```diff
@@ -1,56 +1,69 @@
 import Image from 'next/image';
 import Pillar from '../Pillar/Pillar';
+import {Fade} from "react-awesome-reveal"
 
 export default function MissionPillars() {
   return (
     <section className="flex flex-col w-full md:justify-center items-center bg-white md:bg-[#CEEDF4] py-12 space-y-10">
-      <h2 className="md:uppercase font-bold md:font-light text-2xl md:text-5xl">
+      <h2 className="md:uppercase border-b-2 border-slate-400  font-bold md:font-light text-2xl md:text-5xl">
```
Comment (orama254) [inline src/components/MissionPillars/MissionPillars.tsx]: Remove the additional underline on the h2 to maintain consistency with the initial design.

**Review by orama254 — CHANGES_REQUESTED** [review]
Check out the additional comments for your review.

**Review by orama254 — CHANGES_REQUESTED** [review]
Nice progress @Zackius , 

also remove the underlines from the about us section heading and the events section heading.

See deployed preview for perspective [HERE](https://reactdevske-website-rehjlb39b-react-developers-kenya.vercel.app/)

**Review by orama254 — APPROVED** [review]
Looks Good ✅

**Pull request merged**